src/sentry/static/sentry/app/views/releases/list/releaseCard.tsx

```diff
@@ -35,196 +27,95 @@ type Props = {
-      <PanelBody>
-        <StyledPanelItem>
-          <HeaderLayout>
-            <VersionColumn>
-              <ColumnTitle>{t('Release Version')}</ColumnTitle>
-            </VersionColumn>
-
-            <CreatedColumn>
-              <ColumnTitle>
-                {lastDeploy?.dateFinished ? t('Last Deploy') : t('Date Created')}
-              </ColumnTitle>
-            </CreatedColumn>
-
-            <CommitsColumn>
-              <ColumnTitle>
-                {commitCount > 0
-                  ? [
-                      tn('%s commit', '%s commits', commitCount || 0),
-                      t('by'),
-                      tn('%s author', '%s authors', authors.length || 0),
-                    ].join(' ')
-                  : t('Commits')}
-              </ColumnTitle>
-            </CommitsColumn>
-
-            <NewIssuesColumn>
-              <ColumnTitle>{t('New issues')}</ColumnTitle>
-            </NewIssuesColumn>
-          </HeaderLayout>
-
-          <Layout>
-            <VersionColumn>
-              <VersionWrapper>
-                <Version version={version} tooltipRawVersion truncate anchor={false} />
-              </VersionWrapper>
-            </VersionColumn>
-
-            <CreatedColumn>
-              <TextOverflow>
-                {lastDeploy?.dateFinished && <StyledDeployBadge deploy={lastDeploy} />}
-                <TimeSince date={lastDeploy?.dateFinished || dateCreated} />
-              </TextOverflow>
-            </CreatedColumn>
-
-            <CommitsColumn>
-              <CommitsWrapper>
-                {commitCount > 0 ? (
-                  <ReleaseStats release={release} withHeading={false} />
-                ) : (
-                  <NotAvailable />
-                )}
-              </CommitsWrapper>
-            </CommitsColumn>
-
-            <NewIssuesColumn>
-              <Feature features={['global-views']}>
-                {({hasFeature}) =>
-                  hasFeature ? (
-                    <Tooltip title={t('Open in Issues')}>
-                      <Link to={getReleaseNewIssuesUrl(orgSlug, null, version)}>
-                        <Count value={release.newGroups || 0} />
-                      </Link>
-                    </Tooltip>
-                  ) : (
-                    <Count value={release.newGroups || 0} />
-                  )
-                }
-              </Feature>
-            </NewIssuesColumn>
-          </Layout>
-        </StyledPanelItem>
-      </PanelBody>
-
-      <ReleaseHealth
-        release={release}
-        orgSlug={orgSlug}
-        location={location}
-        showPlaceholders={showHealthPlaceholders}
-        selection={selection}
-      />
+      <ReleaseInfo>
+        <ReleaseInfoHeader>
+          <Version version={version} tooltipRawVersion truncate anchor={false} />
+          {commitCount > 0 ? (
+            <ReleaseStats release={release} withHeading={false} />
+          ) : (
+            <NotAvailable />
```

Comment:
This looks kind of strange that we don't have a label nor anything just `-` (without knowing the context of what's missing).